clients/drcachesim/tools/basic_counts.cpp

```diff
@@ -317,6 +318,7 @@ basic_counts_t::generate_interval_snapshot(uint64_t interval_id)
             snapshot->counters += ctr;
         }
     }
+    snapshot->tid = 0;
```

Comment:
Should this be INVALID_THREAD_ID?

Reply:
I think INVALID_THREAD_ID means something that is not initialized but may be later. But here we just have a convention to set the tid to zero for serial. I'll add a comment instead.